### PR TITLE
Reduce yaml files for textmode

### PR DESCRIPTION
**Pull request by okynos**
We have removed different textmode files to unify in only one.

- Related ticket: https://progress.opensuse.org/issues/163286
- Related MR: https://gitlab.suse.de/qe-yam/openqa-job-groups/-/merge_requests/288
- Needles: N/A
- Verification run:
    Flavor -> Online: https://openqa.suse.de/tests/overview?distri=sle&version=15-SP6&build=93.5&groupid=583
    - x86_64: 
      - 64bit: https://openqa.suse.de/tests/15051541 (Updated)
      - uefi: https://openqa.suse.de/tests/15051540 (Updated)
      - svirt-xen-hvm: https://openqa.suse.de/tests/15051542 (Updated)
      - svirt-xen-pv: https://openqa.suse.de/tests/15051543 (Updated)
    - aarch64: https://openqa.suse.de/tests/15042563 (Updated)
    - ppc64le: https://openqa.suse.de/tests/15051535 (Updated)
    - s390x:
      - s390x-kvm: https://openqa.suse.de/tests/15052040 (Updated)
      - s390x-zVM: https://openqa.suse.de/tests/15052041 (Updated)

### CODE REVIEW

File: schedule/yast/textmode/textmode.yaml [hunk 1]
```diff
@@ -2,41 +2,23 @@
 name: textmode
```
Comment (lemon-suse) [inline schedule/yast/textmode/textmode.yaml:2]: Please move the yaml file to schedule/yam/test_cases/  as a single yaml file.

**Comment by lemon-suse** [comment]
For the VR on xen-pv, https://openqa.suse.de/tests/15001752#, compared with the original job, it missed stop_timeout_system_reboot_now after perform_installation.

File: schedule/yast/textmode/textmode.yaml (hunk 1 above)
Comment (okynos) [reply schedule/yast/textmode/textmode.yaml]: I think it's fixed

**Comment by okynos** [comment]
Solved here https://openqa.suse.de/tests/15052041
Thanks!

**Pull request merged**